src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -205,6 +205,7 @@ public partial class IPAddress
         public static System.Net.IPAddress Parse(string ipString) { return default(System.Net.IPAddress); }
         public override string ToString() { return default(string); }
         public static bool TryParse(string ipString, out System.Net.IPAddress address) { address = default(System.Net.IPAddress); return default(bool); }
+        public long Address { get; set; }
```

Comment:
Not sure about that , this's the desktop message " [ObsoleteAttribute("This property has been deprecated. It is address family dependent. Please use IPAddress.Equals method to perform comparisons. http://go.microsoft.com/fwlink/?linkid=14202")]" . BTW , the link is broken and i am not sure it talks about comparison when it's about setting and getting the Address property